app/_posts/2024-11-05-kuma-2-9-0.md

```diff
@@ -0,0 +1,108 @@
+date: 2024-11-05
+headerImage: /assets/images/blog/test-header.jpg
+tags:
+  - Release
+---
+
+We're excited to announce the biggest release of Kuma since 2.0! 
+In this release we offer a complete beta release of our service rework and namespaced policies.
+
+# Notable features
+In Kuma 2.9.x we've focused on 3 main areas:
+* `MeshTLS` which provides a more granular way to roll out strict Mutual TLS.
+* New resources `MeshService` and `MeshMultiZoneService` which provide a more intuitive and powerful way to manage services in your mesh.
+* A complete rework of our destination policy attachment which works with the new services.
+* Introduction of producer and consumer policies, enabling more flexibility for Kubernetes users.
+
+This set of features greatly simplifies Kuma's user experience, by making it clearer how services exist and by attaching policies
+to specific resources. These changes are also fully integrated with our GUI thus making things incredible easy to understand and troubleshoot.
+
+This release marks an important step in simplifying the Kuma user experience, empowering both key user roles: mesh administrators and service owners.
+
+Feel free to check our [release notes](https://github.com/kumahq/kuma/releases/tag/2.9.0) for the full list of changes.
+
+## New TLS policy
+
+Mutual TLS is one of the most common use cases in Kuma. This is because it's the base on which you can build fully secured  trust environments.
+The most common way to roll out mutual TLS is by first running services in **permissive mode,** in this mode, while the mesh will verify client certificates,
+it will still allow unsecured connection.
+You can then progressively remove unsecure connections by moving clients inside the mesh and once it's complete switch to **strict mode.**
+
+This has been long supported in Kuma. However, it was done at the mesh wide level which created an _all or nothing setup._
+In 2.9.x we've introduced a policy to apply this to only some data planes, thus enabling easier progressive adoption of strong security.
+
+To check how to do this please checkout the [MeshTLS guide](/docs/2.9.x/guides/progressively-rolling-in-strict-mtls/).
+
+This feature is part of a larger effort to address challenges users may be facing rolling out a service mesh in their production environment.
+We are always looking for more input on these challenges, if you are willing to share please feel free to reach out to us on [slack](https://kuma.io/community/).
+
+## New service resources 
+
+Kuma from the beginning have implicitly extracted services from its data planes.
+While this was convenient to get started it was coming with the following shortcomings:
+
+- It wasn't possible to have multiple services fronting the same port of a data plane, this is required for blue-green deployment for example.
+- Sub-setting services was impossible in policies which was making things hard to understand, observe and compose across multiple policies.
```

Comment:
```suggestion
- Creating subsets of services was impossible in policies which made things hard to understand, observe and compose across multiple policies.
```

was it impossible or just cumbersome because of having to repeat tags everywhere?

Reply:
I think just cumbersome